go/polly/README.md

```diff
@@ -0,0 +1,79 @@
+# AWS SDK for Go code examples for Amazon Polly
+
+## Purpose
+
+These examples demonstrate how to perform several Amazon Polly operations.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the _AWS SDK for Go Developer Guide_.
+
+## Running the code
+
+### DescribeVoices/DescribeVoices.go
+
+This example describes the Amazon Polly voices.
+
+`go run DescribeVoices.go`
+
+The unit test mocks the service client and the `DescribeVoices` function.
+
+### ListLexicons/ListLexicons.go
+
+This example lists the Amazon Polly lexicons.
```

Comment:
do they know what lexicons are? Or can we briefly state that here?

Reply:
I've added: , which enable you to customize the pronunciation of words.